Tasks/AppCenterDistributeV3/appcenterdistribute.ts

```diff
@@ -103,7 +103,15 @@ function beginReleaseUpload(apiServer: string, apiVersion: string, appSlug: stri
         "User-Agent": userAgent,
         "internal-request-source": "VSTS"
     };
-    request.post({ url: beginUploadUrl, headers: headers }, (err, res, body) => {
+
+    const requestOptions: request.UrlOptions & request.CoreOptions = { url: beginUploadUrl, headers };
+    if (buildVersion) {
+        requestOptions.body = JSON.stringify({
+            "build_version": buildVersion
```

Comment:
Releases service will ignore this value if it's not required by the app platform, so this may need to be blocked or clarified for users, depending on their app type.

Reply:
Will it fine if we update the helpMarkdown to additionaly have something like "it would took into account only for .zip and .msi binaries, where due to free form nature of distribution appcenter can't resolve the build version automatically"?

Reply:
Maybe "this value will be ignored unless the platform is WPF or winForms"?

Reply:
Yeah thanks, that's sounds better, changed to

The build version of the uploading binary which needs to be specified for `.zip` and `.msi` this value will be ignored unless the platform is WPF or WinForms